pyproject.toml

```diff
@@ -13,45 +14,45 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.11",
 ]
+requires-python = ">=3.11"
+dependencies = [
+    "tagged~=0.0.2",
+    "venusian~=3.1.0",
+    "htm~=0.1.1",
+    "MarkupSafe~=2.1.3",
+    "hopscotch>=0.3.1"
+]
 
-[tool.poetry.urls]
+[tool.hatch.urls]
 Changelog = "https://github.com/pauleveritt/viewdom/releases"
 
-[tool.poetry.dependencies]
-python = "^3.11"
-tagged = "^0.0.2"
-venusian = "^3.1.0"
-htm = "^0.1.1"
-MarkupSafe = "^2.1.3"
-hopscotch = ">=0.3.1"
-
-[tool.poetry.dev-dependencies]
-pytest = "^7.4.3"
-coverage = {extras = ["toml"], version = "^6.3"}
-safety = "^2.3.5"
-mypy = "^1.7.0"
-xdoctest = {extras = ["colors"], version = "^0.15.10"}
-sphinx = "^4.4.0"
-sphinx-autobuild = "^2021.3.14"
-pre-commit = "^3.5.0"
-flake8 = "^6.1.0"
-black = "^22.1"
-flake8-bandit = "^4.1.1"
-flake8-bugbear = "^23.9.16"
-flake8-docstrings = "^1.6.0"
-flake8-rst-docstrings = "^0.3.0"
-pep8-naming = "^0.13.3"
-darglint = "^1.8.1"
-reorder-python-imports = "^3.12.0"
-pre-commit-hooks = "^4.1.0"
-Pygments = "^2.11.2"
-pyupgrade = "^2.31.0"
-nox = "^2023.4.22"
-nox-poetry = "^1.0.3"
-furo = "^2022.3.4"
-myst-parser = "^0.17.0"
-sybil = "^3.0.1"
 
+[tool.hatch.version]
+path = "src/viewdom/version.py"
+
+[tool.hatch.envs.test]
+dependencies = [
+    "pytest~=7.4.3",
+    "sybil~=3.0.1",
+    "coverage[toml]~=6.3",
+    "xdoctest[colors]~=0.15.10",
```

Comment:
xdoctest won't install without distutils (meaning, Python 3.12) and I believe we are dropping this anyway.

Reply:
I just removed